README.md

```diff
@@ -176,6 +176,22 @@ ember install ember-decorators
 ember install @ember-decorators/argument
 ```
 
+## Configuration
+
+You can tweak the following settings in your `config/environment.js` under the `emberDecoratorsArgument` namespace:
+
+### `typeRequired`
+
+**Type**: `Boolean` | **Default**: `false`
+
+If enabled, requires you to also specify a [`@type`](#type) for every [`@argument`](#argument).
```

Comment:
Should probably note on this that it could break addons - If an addon chooses to just mark arguments without types, and the user turns this on, there's no way for us to know which components should/shouldn't have required types. I haven't been able to think of a solution for that one yet, all ears if you have any ideas